src/TruncatedAccount/index.stories.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import TruncatedAccount from "./";
+
+const Template = (args) => <TruncatedAccount {...args} />;
+
+export const Overview = Template.bind({});
+Overview.args = {
+  name: "Secondary Checking Account",
+  lastFour: 4321,
+};
+
+export const LongAccountName = (args) => (
+  <>
+    <h4 className="margin--bottom">The element below can be resized:</h4>
+    <div
+      style={{
+        outline: "1px dotted hotpink",
+        width: "300px",
+        height: "36px",
+        resize: "both",
+        overflow: "auto",
+      }}
+    >
+      <TruncatedAccount {...args} />
+    </div>
+  </>
+);
+LongAccountName.args = {
+  name: "Secdonary Savings Account / Car Repair Funds",
```

Comment:
minor typo here (I know it's only an example but still)

Reply:
thanks!